### ISSUE TITLE
Bug: Fresh Fiji on Win7 does not work with insight-ij

### ISSUE THREAD

**Issue by pwalczysko** [ome/omero-insight#70]
See https://github.com/ome/omero-insight/pull/65#issuecomment-523416136. 

Repasting the error (which comes after you logged in, and the splashscreen vanishes).

```
[Wed Aug 21 12:21:12 BST 2019] [INFO] [] Reading available sites from https://imagej.net/
[Wed Aug 21 12:23:10 BST 2019] [ERROR] [] null
java.lang.NoSuchMethodException: Could not find method com.sun.jna.Native.load(Ljava/lang/String;Ljava/lang/Class;Ljava/util/Map;)Lcom/sun/jna/Library;
There was a problem with the class com.sun.jna.Native which can be found here:
/C:/Users/petr/DOWNLO~1/FIJI-W~1/Fiji.app/jars/jna-4.5.2.jar
/C:/Users/petr/DOWNLO~1/FIJI-W~1/Fiji.app/plugins/OMERO.imagej-5.5.4-SNAPSHOT/lib/jna-5.3.0.jar

WARNING: multiple locations found!
java.lang.NoSuchMethodError: com.sun.jna.Native.load(Ljava/lang/String;Ljava/lang/Class;Ljava/util/Map;)Lcom/sun/jna/Library;
	at com.sun.jna.platform.win32.Version.<clinit>(Version.java:37)
	at org.openmicroscopy.shoola.env.data.model.appdata.WindowsApplicationDataExtractor.extractAppData(WindowsApplicationDataExtractor.java:236)
	at org.openmicroscopy.shoola.env.data.model.ApplicationData.<init>(ApplicationData.java:112)
	at org.openmicroscopy.shoola.agents.treeviewer.view.TreeViewerFactory.readExternalApplications(TreeViewerFactory.java:477)
	at org.openmicroscopy.shoola.agents.treeviewer.view.TreeViewerFactory.getTreeViewer(TreeViewerFactory.java:427)
	at org.openmicroscopy.shoola.agents.treeviewer.view.TreeViewerFactory.getTreeViewer(TreeViewerFactory.java:193)
	at org.openmicroscopy.shoola.agents.treeviewer.TreeViewerAgent.activate(TreeViewerAgent.java:661)
	at org.openmicroscopy.shoola.env.Container.activateAgents(Container.java:327)
	at org.openmicroscopy.shoola.env.Container.startService(Container.java:360)
	at org.openmicroscopy.shoola.env.init.Initializer.notifyEnd(Initializer.java:197)
	at org.openmicroscopy.shoola.env.Container.startupInPluginMode(Container.java:488)
	at org.openmicroscopy.shoola.env.Container.startupInPluginMode(Container.java:412)
	at org.openmicroscopy.shoola.MainIJPlugin.run(MainIJPlugin.java:272)
	at ij.IJ.runUserPlugIn(IJ.java:230)
	at ij.IJ.runPlugIn(IJ.java:193)
	at ij.Executer.runCommand(Executer.java:137)
	at ij.Executer.run(Executer.java:66)
	at java.lang.Thread.run(Thread.java:748)

	at com.sun.jna.platform.win32.Version.<clinit>(Version.java:37)
	at org.openmicroscopy.shoola.env.data.model.appdata.WindowsApplicationDataExtractor.extractAppData(WindowsApplicationDataExtractor.java:236)
	at org.openmicroscopy.shoola.env.data.model.ApplicationData.<init>(ApplicationData.java:112)
	at org.openmicroscopy.shoola.agents.treeviewer.view.TreeViewerFactory.readExternalApplications(TreeViewerFactory.java:477)
	at org.openmicroscopy.shoola.agents.treeviewer.view.TreeViewerFactory.getTreeViewer(TreeViewerFactory.java:427)
	at org.openmicroscopy.shoola.agents.treeviewer.view.TreeViewerFactory.getTreeViewer(TreeViewerFactory.java:193)
	at org.openmicroscopy.shoola.agents.treeviewer.TreeViewerAgent.activate(TreeViewerAgent.java:661)
	at org.openmicroscopy.shoola.env.Container.activateAgents(Container.java:327)
	at org.openmicroscopy.shoola.env.Container.startService(Container.java:360)
	at org.openmicroscopy.shoola.env.init.Initializer.notifyEnd(Initializer.java:197)
	at org.openmicroscopy.shoola.env.Container.startupInPluginMode(Container.java:488)
	at org.openmicroscopy.shoola.env.Container.startupInPluginMode(Container.java:412)
	at org.openmicroscopy.shoola.MainIJPlugin.run(MainIJPlugin.java:272)
	at ij.IJ.runUserPlugIn(IJ.java:230)
	at ij.IJ.runPlugIn(IJ.java:193)
	at ij.Executer.runCommand(Executer.java:137)
	at ij.Executer.run(Executer.java:66)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by pwalczysko** [comment]
cc @jburel

**Comment by jburel** [comment]
Fixed in #72
